pom.xml

```diff
@@ -109,7 +109,7 @@
         <config-version>1.3</config-version>
         <ft-version>2.0</ft-version>
         <health-version>2.1</health-version>
-        <metrics-version>2.1.0</metrics-version>
+        <metrics-version>2.2-RC1</metrics-version>
```

Comment:
Now that Metrics 2.2 released, then this should be 2.2

Reply:
Whups, email reply didn't apply to this thread. Yeah, I agree.

Reply:
So, skip the RC and go straight to final?

Reply:
Agreed